### ISSUE TITLE
Problem with Training VGG Net

### ISSUE THREAD

**Issue by PetrusKe** [Microsoft/CNTK#1544]
Hi, 
I try to train official **_VGG_** example in cntk, but it seems not work well.

So I checked the training setting in VGG Net in paper, and I find their parameters configuration for training:
**(in caffe)batchsize=256, base_lr=0.01, parallelGPUs=4**
Transform to cntk, I think base_lr need to be setted 0.1

However, in cntk official example, **batchsize=128 and base_lr=0.1**
Is the change in the relationship between batchsize and learning rate causes my training problem?
( lr is too big? )

Thanks.

**Comment by cha-zhang** [comment]
Sorry we used to have some memory issue with CNTK and has to choose a smaller batchsize. Recently we have done a lot of work in that regard and the problem has been solved. However, we haven't retrained the VGG network ourselves. 

We will do an internal training and update the official example.

**Comment by cha-zhang** [comment]
The Python example https://github.com/Microsoft/CNTK/blob/master/Examples/Image/Classification/VGG/Python/VGG16_ImageNet_Distributed.py should work now. Thanks!
